openIMIS/openIMIS/git_scripts.py

```diff
@@ -0,0 +1,112 @@
+import git
+import json
+import os
+
+
+from os import path
+from django.conf import settings
+from openIMIS.openimisapps import openimis_apps
+
+
+def create_release_branches_backend(version):
+    """
+       function to create release branches 'release/<version>'
+       for all backend modules presented in openimis.json
+    """
+    output_messages = []
+    # modules = openimis_apps()
+    # TODO temporary value for modules for test
+    modules = ['calculation_rule-fs_income_percentage', 'report', 'policyholder', 'contribution_plan', 'calculation']
+    directory = path.abspath(path.join(settings.BASE_DIR, "../.."))
+    release_branch = f'release/{version}'
+    for module in modules:
+        try:
+            repo_name = f'openimis-be-{module}_py'
+            # check if repo exist - if no - clone to local
+            if not path.exists(f'{directory}/{repo_name}'):
```

Comment:
I think it'll be better to use `listdir()` on `directory` and check if repo_name is in there instead of checking existence of string path.